api/samples/CMakeLists.txt

```diff
@@ -172,6 +172,20 @@ function (add_sample_client name source_file_list extension_list)
     "${sample_list};${name}")                                     # NON-PUBLIC
   set(srcs ${srcs} ${source_file_list} PARENT_SCOPE)              # NON-PUBLIC
   set(tgts ${tgts} ${name} PARENT_SCOPE)                          # NON-PUBLIC
+
+  # Some samples actually have more tests than just simple sanity tests. These
```

Comment:
As mentioned before, this should all be hidden, so `# NON-PUBLIC`

Reply:
Do you mean, I start the comment with NON-PUBLIC?

Reply:
Not sure what the script stripping these looks for precisely: whatever works for it.  See how the lines above have that marker?  Some CMake code strips such lines out when coping this for export (yeah kind of hacky).

Reply:
I would think the same suffix would look best and should match the script.

Reply:
Done.